asset_bar_op.py

```diff
@@ -1503,6 +1504,54 @@ def handle_key_input(self, event):
                     asset_type=asset_data["assetType"],
                 )
             return True
+        elif (
+            event.type == "V"
+            and event.shift
+            and self.active_index > -1
+            and utils.profile_is_validator()
+        ):
+            sr = global_vars.DATA["search results"]
+            asset_data = sr[self.active_index]
+            bpy.ops.object.blenderkit_change_status(
+                asset_id=asset_data["id"], state="validated"
+            )
+            return True
+        elif (
+            event.type == "H"
+            and event.shift
+            and self.active_index > -1
+            and utils.profile_is_validator()
+        ):
+            sr = global_vars.DATA["search results"]
+            asset_data = sr[self.active_index]
+            bpy.ops.object.blenderkit_change_status(
+                asset_id=asset_data["id"], state="on_hold"
+            )
+            return True
+        elif (
+            event.type == "U"
+            and event.shift
+            and self.active_index > -1
+            and utils.profile_is_validator()
+        ):
+            sr = global_vars.DATA["search results"]
+            asset_data = sr[self.active_index]
+            bpy.ops.object.blenderkit_change_status(
+                asset_id=asset_data["id"], state="uploaded"
+            )
+            return True
+        elif (
```

Comment:
Nested `elif`s are not neccesserary and make the code less readable and more complicated.